File: Android/Tejash11_TejashSeth_2024csit1065_2/Patterns/Pattern5.java
import java.util.*;

public class Main {

  public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int space = n / 2 + 1, star = 1;
    for (int i = 1; i <= n; i++)
    {
      for (int j = 1; j < space; j++)
      {
        System.out.print("\t");
      }
      for (int j = 1; j <= star; j++)
      {
        System.out.print("*\t"); 
      }
      for (int j = 1; j <= space; j++)
      {
        System.out.print("\t");  
      }

      if (i <= n / 2)
      {
        star += 2;
        space--;
      }
      else
      {
        space++;
        star -= 2;
      }
      System.out.println();
    }
  }
}

File: AR-VR/vanshverma721_VanshVerma_2024csit1153_2/pattern/pattern_5.cpp
#include <iostream>
using namespace std;

int main()
{
 int i,j,n,space,star;
   scanf("%d",&n);
   space=n/2;
   star=1;
   for(i=1;i<=n;i++)
   {
       for(j=1;j<=space;j++)
       {
          printf("\t");
           
       }
       for(j=1;j<=star;j++)
       {
           printf("*\t");
       }
       
       if(i<=n/2)
       {
           space--;
           star=star+2;
       }
       else
       {
           space++;
           star=star-2;
       }
       printf("\n");
   }
}

   

    
